packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -868,8 +868,17 @@ mixin WidgetInspectorService {
     registerServiceExtension(
       name: name,
       callback: (Map<String, String> parameters) async {
-        if (parameters.containsKey('enabled'))
-          await setter(parameters['enabled'] == 'true');
+        if (parameters.containsKey('enabled')) {
+          final bool value = parameters['enabled'] == 'true';
+          await setter(value);
+          developer.postEvent(
```

Comment:
widget_inspector already has a postEvent member you can use to make this easy to test.

Reply:
done.